app/services/clients/adminusers_client.js

```diff
@@ -567,23 +567,21 @@ module.exports = function (clientOptions = {}) {
   const updateServiceName = (serviceExternalId, serviceName, serviceNameCy) => {
     const body = []
     if (serviceName) {
```

Comment:
If this service is always going to be there (because it is populated before this call, we can remove this check.
The default is 'system generated' so we can put that in here, as `value: serviceName || 'System Generated'` so that it is immediately clear to whoever reads the code what's going on with the service name